packages/at_client/lib/src/util/sync_util.dart

```diff
@@ -117,7 +118,7 @@ class SyncUtil {
     if (key.startsWith(AT_PKAM_PRIVATE_KEY) ||
         key.startsWith(AT_PKAM_PUBLIC_KEY) ||
         key.startsWith(AT_ENCRYPTION_PRIVATE_KEY) ||
-        key.startsWith(statsNotificationId)) {
+        key.startsWith(NotificationServiceImpl.lastReceivedNotificationKey)) {
```

Comment:
Just in case, please exclude the old key name also, and change this to 
```
        key.startsWith(NotificationServiceImpl.notificationIdKey) ||
        key.startsWith(NotificationServiceImpl.lastReceivedNotificationKey)) {
```